src/main/scala/uk/gov/hmrc/SbtSassCompiler.scala

```diff
@@ -120,7 +120,16 @@ object SbtSassCompiler extends AutoPlugin {
             (anySassFileChanged: Boolean, _) =>
               if (anySassFileChanged || (sassFilesFound.nonEmpty && existingCssFromTargetDirectory.isEmpty)) {
                 logger.info("sbt-sass-compiler: Uncompiled Sass files found")
-                compileSassToCssFiles()
+                try {
+                  compileSassToCssFiles()
+                } catch {
+                  case ise: IllegalStateException => {
+                    logger.error("sbt-sass-compiler: Sass compilation failed: " + ise)
+                    logger.info("sbt-sass-compiler: attempting to recover by removing the path and re-running SassCompilerFactory.bundled()")
+                    Files.delete(Path.of(s"/tmp/de.larsgrefer.sass.embedded.connection.BundledPackageProvider/4.0.2/dart-sass/1.83.4/dart-sass"))
```

Comment:
With this, one thing I just thought of - I thought that Files.delete wouldn't recursively delete a directory

Might be worth using [sbt.io.IO.delete(File)](https://www.scala-sbt.org/1.x/api/sbt/io/IO$.html#delete(file:java.io.File):Unit) which does delete recursively if it's a directory

And checking if the sbt-test is only removing the sass executable